### PR TITLE
Adjust query

### PR DESCRIPTION
**Pull request by jxb345**
- adjusted db query to grab book details of each of the maximum books by the author
- made indentation fixes to CSS in React components
- added additional CSS styling to components
- added an author_image and cover_image fields to authors and books tables
- added sequelize-cli (to address prior issuing of creating the db

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ class App extends React.Component {
 
   getInfo() {
     $.ajax({
-      url: '/author',
+      url: 'http://localhost:3002/author',
```
Comment (MuhammadJamaluddin) [inline client/src/components/App.jsx:26]: good fix

File: client/src/components/About.jsx [hunk 2]
```diff
@@ -1,18 +1,17 @@
 import React from 'react';
 import styled from 'styled-components';
-import $ from 'jquery';
 
 const Paragraph = styled.p`
```
Comment (MuhammadJamaluddin) [inline client/src/components/About.jsx]: I would name the component a more descriptive name

File: client/src/components/About.jsx [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
-font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif;
-height: 24px;
-width: auto;
-font-weight: bold;
-font-size: 12px;
-text-transform: uppercase;
-padding: 2px 0;
-border-bottom: 1px solid #D8D8D8;
-`
+  color: #382110;
```
Comment (MuhammadJamaluddin) [inline client/src/components/About.jsx:5]: I would name the component a more descriptive name

File: client/src/components/AuthorName.jsx [hunk 4]
```diff
@@ -1,25 +1,32 @@
...
-font-family: "Merriweather", "Georgia", serif;
-font-size: 16px;
-font-weight: bold;
-`
-
+  font-family: "Merriweather", "Georgia", serif;
+  font-size: 16px;
+  font-weight: bold;
+`;
 const Span = styled.span`
```
Comment (MuhammadJamaluddin) [inline client/src/components/AuthorName.jsx]: the same naming suggestion

File: client/src/components/AuthorName.jsx [hunk 5]
```diff
@@ -1,25 +1,32 @@
...
-color: #999999;
-margin-top: 4px;
-margin-bottom: 4px;
+  color: #999999;
```
Comment (MuhammadJamaluddin) [inline client/src/components/AuthorName.jsx:10]: the same naming suggestion

File: chai.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import chai from 'chai';
+import chaiEnzyme from 'chai-enzyme';
+
+chai.use(chaiEnzyme());
```
Comment (MuhammadJamaluddin) [inline chai.js:4]: Note sure what this file is for

Comment (jxb345) [reply chai.js:4]: Adding chai allowed me to use the 'expect' statements w/in Enzyme. I know I *shouldn't* have to add chai, but when I added it - the error messages went away.

**Pull request merged**